testhelpers/helpers.go

```diff
@@ -1,6 +1,20 @@
 package testhelpers
 
+import (
+	"fmt"
+	"net/http"
+)
+
 // Intp returns a pointer to the given integer value.
 func Intp(i int) *int {
 	return &i
 }
+
+// MustNewRequest creates a new http get request or panics if it can't
+func MustNewRequest(rawurl string) *http.Request {
```

Comment:
The method should match the parameters of `http.NewRequest`, i.e., accept a method and body parameter that would then be passed through.